Services/BookingService.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Airport_Ticket_Booking_System.Models;
+using Airport_Ticket_Booking_System.Models.Enums;
+using Airport_Ticket_Booking_System.Repositories;
+
+namespace Airport_Ticket_Booking_System.Services;
+
+
+public static class BookingService
+{
+    public static void BookFlight(User user, string userAnswer)
```

Comment:
It doesn't make sense to use userAnswer here ...
```suggestion
    public static void BookFlight(User user, string flightIdStr)
```